src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -105,7 +105,14 @@ def put(self, request, group, integration_id):
 
         # TODO(jess): would be helpful to return serialized external issue
         # once we have description, title, etc
-        return Response(status=201)
+        url = data.get('url') or ''
+        context = {
+            'id': external_issue.id,
+            'key': external_issue.key,
+            'url': url,
+            'integrationId': external_issue.integration_id,
+        }
+        return Response(context, status=201)
```

Comment:
i think this is fine for now and we can follow up by moving this into a serializer